eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -179,8 +179,6 @@ public void testAssignments() {
     public void testErrorGracefulModeOn() {
         try {
             initClient(TEST_HOST, false, true, true);
-            System.out.println("Sleeping for a bit to wait for cache population to complete");
```

Comment:
Don't need these sleeps since we are no formally waiting for the request to complete.